_laws/boyds_law_of_iteration.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Boyd's Law of Iteration
+law: Speed of iteration beats quality of iteration
+law-author: John Boyd
+law-url: https://blog.codinghorror.com/boyds-law-of-iteration/
+law-year: Around 1987
```

Comment:
I think it was more like the 50s right? Seems as though there is no definitive date?